### DISCUSSION TITLE
[QUESTION] Is there a VS visualizer for SkiaSharp?

### DISCUSSION THREAD

**Discussion by CpBobette** [mono/SkiaSharp#2888] [Show and tell]
Is there a visualizer for Visual Studio that allows to inspect images during debug (like the one available from OpenCv)?

**Comment by mattleibow** [comment]
This is a very good question. No, for now. But definitely something that will be very helpful.

Are you talking about this one: https://marketplace.visualstudio.com/items?itemName=VisualCPPTeam.ImageWatch

**Comment by mlptownsend** [comment]
If anyone is out there looking for something like this, I created a debugger visualizer for SKBitmap, SKImage, and SKSurface.
https://github.com/MapLarge/SkiaSharpVisualizer

**Reply by mattleibow** [reply]
That looks amazing!
